sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
There are several ways an AlgebraicNumber is constructed in ``__new__``. This is the last one, the default. Its content is obvious from the line
```
rep = DMP.from_list([1, 0], 0, dom)
```
which gives the actual coefficients to be used. Only the Tuple is not updated accordingly, most probably forgotten as it only appears in the tests. The alternative constructions use the same set  of coefficients for both ``rep`` and ``scoeffs``.

(The coefficients are currently in two places, ``rep`` and ``scoeffs``. Those that are actually used in computations are in ``rep`` while the other set only appears in the tests.) 

Changing the default of  ``coeffs`` is problematic because it is used in
```
if coeffs != Tuple():
```
to indicate that coefficients were included in the call.

The sign treatment is something I would rather do without. It is only meaningful for negative real numbers. But since it was there, I decided to leave it.